Packs/CortexResponseAndRemediation/Playbooks/silent-Playbook-Azure_AD_Administrator_Role_Assignment.yml

```diff
@@ -0,0 +1,2813 @@
+      - "38"
+    scriptarguments:
+      key:
+        simple: Evidence.AssignorUser.DefenderAlerts
+      value:
+        complex:
+          root: Microsoft365Defender.Hunt.results
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Microsoft365Defender.Hunt.results.AccountUpn
+                iscontext: true
+              right:
+                value:
+                  simple: InvestigationUsers.Assignor.UPN
+                iscontext: true
+              ignorecase: true
+          accessor: Title
+          transformers:
+          - operator: uniq
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ' | '
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1993,
+          "y": 1655
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "63":
+    id: "63"
+    taskid: 6076f260-beee-439e-b564-8c2be5621e5e
+    type: regular
+    task:
+      id: 6076f260-beee-439e-b564-8c2be5621e5e
+      version: -1
+      name: Save Assignor-Triggered Detections
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "38"
+    scriptarguments:
+      key:
+        simple: Evidence.AssignorUser.AzureRiskDetections
+      value:
+        complex:
+          root: MSGraph.identityProtection.risks.riskDetections
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: MSGraph.identityProtection.risks.riskDetections.userPrincipalName
+                iscontext: true
+              right:
+                value:
+                  simple: InvestigationUsers.Assignor.UPN
+                iscontext: true
+              ignorecase: true
+          accessor: riskEventType
+          transformers:
+          - operator: uniq
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ' | '
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1512,
+          "y": 1655
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "64":
+    id: "64"
+    taskid: 6acab5cf-776d-44d1-8d6c-e7c2cd299f11
+    type: regular
+    task:
+      id: 6acab5cf-776d-44d1-8d6c-e7c2cd299f11
+      version: -1
+      name: Save Assignor-Triggered Detections
+      description: where the **risk state** is either *atRisk* or *confirmedCompromised* and the **risk level** is *medium* or *high*
```

Comment:
Where the **risk state** is either *atRisk* or *confirmedCompromised* and the **risk level** is *medium* or *high*